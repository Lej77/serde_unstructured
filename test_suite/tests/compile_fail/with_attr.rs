
#[derive(serde_unstructured::SerdeView)]
struct Data {
    #[serde(with = "path")]
    field: u8,
}

fn main() {}