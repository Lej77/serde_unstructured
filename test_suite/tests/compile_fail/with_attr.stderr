error: SerdeView can't project into fields that are using custom deserialization, skip the field manually using a `#[serde_view(skip)]` attribute or use an empty `#[serde_view()]` attribute to ignore any `#[serde]` attributes.
 --> tests/compile_fail/with_attr.rs:5:5
  |
5 |     field: u8,
  |     ^^^^^
